config/application.rb

```diff
@@ -27,7 +27,6 @@ class Application < Rails::Application
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{rb,yml}')]
     config.i18n.default_locale = :nl
-    config.i18n.fallbacks = true
```

Comment:
Why would we want to remove fallbacks? It's better to show a text in the 'wrong' language instead of a translation error in my opinion

Reply:
This was from the original PR I think. This line was removed, since the line below it defines the fallbacks. The true here is not correct, the below config is